uber/templates/registration/new.html

```diff
@@ -9,72 +11,76 @@
 
 <script type="text/javascript">
     // automatically log out after 15 minutes of inactivity
-    setTimeout(function() {
+    setTimeout(function () {
         window.location = "../accounts/logout";
     }, 900000);
     
     var UNASSIGNED = {{ unassigned|jsonize }};
     var groupChosen = function(group) {
         var $group = $(group);
-        var $link = $group.prev("a");
-        var $badgeNum = $group.nextAll(":text");
-        if ($group.val() == "") {
+        var $link = $group.prev('a');
+        var $badgeNum = $group.nextAll(':text');
+        if ($group.val() === '') {
             $link.hide();
```

Comment:
Hmm, I'm of two minds on this.

One the one hand, you're totally right that mixing huge swaths of style changes alongside bugfixes and functionality changes makes it super difficult for people reading the code reviews to notice which is which.

On the other hand, we have a ton of legacy stuff which is using outdated coding, styling, and formatting practices, and if we don't update that stuff while we're working on these files anyway it'll probably never get done.

How about this as a compromise: anytime I make a change that mixes styling and functionality changes, I will leave a comment on literally every place where there's a functionality change and/or a bugfix.  Similarly, I will leave a comment on literally every place that's merely formatting/styling so that there's no ambiguity.  I don't mind doing that, since I'm already in the habit of commenting on pretty much everything I'm doing anyway so this isn't even much extra effort and is probably good practice anyway.  Plus it does give my a good chance/place to make our formatting consistent over time.

Does that sound good?